sql/plan/cross_join.go

```diff
@@ -75,6 +75,13 @@ func (p *CrossJoin) TransformExpressionsUp(f func(sql.Expression) (sql.Expressio
 	return NewCrossJoin(left, right), nil
 }
 
+func (p *CrossJoin) String() string {
+	pr := sql.NewTreePrinter()
+	_ = pr.WriteNode("CrossJoin")
```

Comment:
Should we print the conditions too?

Reply:
CrossJoin does not have conditions (at least ours doesn't)

Reply:
oops, sorry.